thorlcr/master/thmastermain.cpp

```diff
@@ -654,7 +654,7 @@ int main( int argc, const char *argv[]  )
             try
             {
                 LOG(MCdebugProgress, thorJob, "calling initClientProcess %d", thorEp.port);
-                initClientProcess(serverGroup, DCR_ThorMaster, thorEp.port);
+                initClientProcess(serverGroup, DCR_ThorMaster, thorEp.port, NULL, NULL, MP_WAIT_FOREVER, true);
```

Comment:
not sure master has to listen?

Reply:
master yes, but the slaves no, they create a new MP server.  I will change that in slavmain.cpp